UI/src/stories/molecules/RipaOfficer.stories.js

```diff
@@ -0,0 +1,18 @@
+import RipaStopReason from '@/components/molecules/RipaOfficer'
+
+export default {
+  title: 'Molecules/RipaOfficer',
+  component: RipaOfficer,
+  parameters: {},
+}
+
+export const basic = () => ({
+  components: { RipaOfficer },
+  data() {
+    return {
+      officerYears
```

Comment:
I am getting an error that officer is not defined. Probably need to add: 

`officer: {}`